src/assets/css/index.css

```diff
@@ -0,0 +1,460 @@
+* {
```

Comment:
- [ ] In the mobile view, the content overflows the navbar, even if I open it:

This happens on:

Reserve car:

![image](https://github.com/Chandan-devs-tech/drive-easy-frontend/assets/57421823/a98c6d53-d24b-46bf-b7be-cf54ccfffa5d)

Even if I open the navbar, it won't let me click on it or navigate to any link. 

Kindly, fix this issue so your app works as intended . 👍